web/views/layout.erb

```diff
@@ -36,6 +36,9 @@
             <ul class="nav navbar-nav">
               <li><a href="<%= url '/' %>">Execution plans</a></li>
               <li><a href="<%= url '/status' %>">Status</a></li>
+              <% if back %>
+                  <li><a href="<%= back %>">Back to Foreman</a></li>
```

Comment:
We could, but there are 2 problems:
1) User comes to dynflow console through the link in foreman_tasks list. Then clicks on anything in navbar and has no way of getting back to the foreman since `request.referrer` now points to task show page in dynflow.
2) Accessing the task in dynflow by url makes it point to itself. This could be possible source of confusion because 'Back to Foreman' now just refreshes the page.